src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
+@import "./header.scss";
+@import "./card.scss";
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --background-color: #fff;
+  --second-background-color: #00acdc;
+  --text-color: #060b35;
+  --second-text-color: #616070;
+  --border-color: #f3f3f3;
+  --color: #00acdc;
   margin: 0;
 }
+
+.header {
+  text-transform: uppercase;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 50px 40px;
+    align-self: center;
+  }
+
+  &__item:nth-child(n) {
+    margin-bottom: 46px;
+  }
+
+  &__item:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
+
+@media (max-width: 1023px) {
+  .main__container {
+    padding: 50px 0;
+  }
+}
+
+@media (min-width: 1200px) {
+  .main__container {
+    padding: 50px 128px;
+  }
+}
```

Comment:
Not sure you need it, anyway:
you can write media queries directly in `&__container` selector, for example:

```
&__container {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  padding: 50px 40px;
  align-self: center;
    
  @media (min-width: 1200px) {
    padding: 50px 128px;
  }
}
```
Just FYI